frontend/src/app/admin/admin.module.ts

```diff
@@ -22,6 +22,9 @@ import {OperateurService} from './services/operateurs/operateur.service';
 import { AdminTypeRequetesComponent } from './components/admin-content/admin-type-requetes/admin-type-requetes.component';
 import { AdminTypeRequetesFormComponent } from './components/admin-content/admin-type-requetes-form/admin-type-requetes-form.component';
 import {TypeRequeteService} from "./services/type-requetes/type-requete.service";
+import { AdminMotifComponent } from './components/admin-content/admin-motif/admin-motif.component';
+import {MotifService} from './services/motifs/motif.service';
```

Comment:
The service should be named as MotifSuppressionService.

Please do not forget to update the file names